mathics/doc/common_doc.py

```diff
@@ -805,7 +841,7 @@ def __init__(
 
         # Needs to come after self.chapter is initialized since
         # XMLDoc uses self.chapter.
-        self.xml_doc = DocumentationEntry(text, title, self)
+        self.doc = DocumentationEntry(text, title, self)
```

Comment:
@rocky, we agree that "xml" is misleading, right?

Reply:
Yes, it is misleading or just plain wrong. 

It is possible (I hope) in the future we will hook into Sphinx's representation of a documentation object.